debian/control

```diff
@@ -404,6 +418,7 @@ Architecture: linux-any
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${misc:Depends},
+         mir-platform-graphics-atomic-kms,
```

Comment:
Hm. But you haven't actually *added* a `mir-platform-graphics-atomic-kms` package, just a `mir-platform-graphics-atomic-kms22` package.

Maybe what we should do is add `mir-platform-graphics-atomic-kms22` to the dependencies of the `mir-platform-graphics-gbm-kms` metapackage?

Reply:
And then we'll have a `Provides:` / `Breaks:` / `Replaces:`, when we want to move over?

We should probably add `-atomic-kms` already anyway.

Reply:
> We should probably add `-atomic-kms` already anyway.

Done.